[2024-01-05T21:00:25.797+0300] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Google_X_mention_score.process_data scheduled__2024-01-01T01:30:00+00:00 [queued]>
[2024-01-05T21:00:25.808+0300] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Google_X_mention_score.process_data scheduled__2024-01-01T01:30:00+00:00 [queued]>
[2024-01-05T21:00:25.809+0300] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2024-01-05T21:00:25.809+0300] {taskinstance.py:1289} INFO - Starting attempt 1 of 2
[2024-01-05T21:00:25.809+0300] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2024-01-05T21:00:25.821+0300] {taskinstance.py:1309} INFO - Executing <Task(BashOperator): process_data> on 2024-01-01 01:30:00+00:00
[2024-01-05T21:00:25.823+0300] {standard_task_runner.py:55} INFO - Started process 59782 to run task
[2024-01-05T21:00:25.827+0300] {standard_task_runner.py:82} INFO - Running: ['airflow', 'tasks', 'run', 'Google_X_mention_score', 'process_data', 'scheduled__2024-01-01T01:30:00+00:00', '--job-id', '17', '--raw', '--subdir', 'DAGS_FOLDER/google_mention_score.py', '--cfg-path', '/tmp/tmpgvgs9j2i']
[2024-01-05T21:00:25.828+0300] {standard_task_runner.py:83} INFO - Job 17: Subtask process_data
[2024-01-05T21:00:25.931+0300] {task_command.py:389} INFO - Running <TaskInstance: Google_X_mention_score.process_data scheduled__2024-01-01T01:30:00+00:00 [running]> on host prod-srv
[2024-01-05T21:00:26.022+0300] {taskinstance.py:1516} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=admin
AIRFLOW_CTX_DAG_ID=Google_X_mention_score
AIRFLOW_CTX_TASK_ID=process_data
AIRFLOW_CTX_EXECUTION_DATE=2024-01-01T01:30:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-01-01T01:30:00+00:00
[2024-01-05T21:00:26.023+0300] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2024-01-05T21:00:26.024+0300] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python3 /home/igor/mlops_4/scripts/process_data.py']
[2024-01-05T21:00:26.029+0300] {subprocess.py:86} INFO - Output:
[2024-01-05T21:00:26.434+0300] {subprocess.py:97} INFO - Command exited with return code 0
[2024-01-05T21:00:26.478+0300] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=Google_X_mention_score, task_id=process_data, execution_date=20240101T013000, start_date=20240105T180025, end_date=20240105T180026
[2024-01-05T21:00:26.522+0300] {local_task_job.py:212} INFO - Task exited with return code 0
[2024-01-05T21:00:26.552+0300] {taskinstance.py:2596} INFO - 1 downstream tasks scheduled from follow-on schedule check
